sep-requirements.txt

```diff
@@ -0,0 +1,2 @@
+pre-commit==1.17.0
```

Comment:
Usually, you want to include runtime dependencies too.

Reply:
What do you mean?

Reply:
Sorry, I did not quite understand.

Reply:
You have Django as a runtime dependency. Usually, tests require such deps too. But I guess this can be omitted for now.